include/grpc/grpc.h

```diff
@@ -351,6 +351,19 @@ typedef struct grpc_op {
   } data;
 } grpc_op;
 
+/* Propagation bits: this can be bitwise or-ed to form propagation_mask for
+ * grpc_call */
+/** Propagate deadline */
+#define GRPC_PROPAGATE_DEADLINE 1
```

Comment:
This is a style/personal preference thing, but I think it is cleaner to put these in an enum, e.g.
/** Propagation bits: these can be bitwise or-ed to form a propagation_mask for grpc_call(). */
enum grpc_propagation_bits {
  GRPC_PROPAGATION_NONE = 0 /* Do not propagate */
  GRPC_PROPAGATION_DEADLINE = 1 /* Propagate deadline */
  ...
};